packages/theming/atlas/tests/e2e/allScreens.spec.ts

```diff
@@ -68,10 +49,6 @@ describe("Screenshots of the pages for", () => {
             });
         }
     }
-
-    afterAll(() => {
```

Comment:
Why did we removed this ? I cannot recall exactly why i added this option but if we going to remove this maybe remove the `cleanUnusedScreenshotBases` function aswell since this is the only place it is being used.

Reply:
Yeah, I removed the function as well.